.circleci/unittest/windows/scripts/install.sh

```diff
@@ -43,6 +43,9 @@ if [ ! -z "${CUDA_VERSION:-}" ] ; then
 fi
 
 # 2. Install torchaudio
+printf "* Installing fsspec\n"
+pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org fsspec
```

Comment:
Q: Which dependency requires this?

Reply:
`pytorch-lightning` depends on fsspec. Installing `pytorch-lightning` fails on windows in the GHA env due to SSL cert issues, but this fixes the issue.